src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -179,10 +179,14 @@ private void setBindingsDriver() {
     new JoystickButton(driverController, Driver.resetFieldOrientationButton)
         .onTrue(new InstantCommand(drivetrain::resetFieldOrientation));
     // axisTrigger(driverController, Axis.kRightTrigger)
-    // .whileTrue(new SequentialCommandGroup(new PrintCommand("Running Intake"),
-    // new AutoMATICALLYGetNote(drivetrain, intakeShooter, limelight)));
-    new POVButton(driverController, 0).whileTrue(
-        new AutoMATICALLYGetNote(drivetrain, intakeShooter, limelight));
+    //     .whileTrue(new SequentialCommandGroup(new PrintCommand("Running Intake"),
+    //         new AutoMATICALLYGetNote(drivetrain, intakeShooter, limelight)));
+    
+    new POVButton(driverController, 0)
+            .whileTrue(new ParallelCommandGroup(new Intake(intakeShooter),
+                    new AutoMATICALLYGetNote(drivetrain, limelight,
+                            intakeShooter)));
```

Comment:
does the command still need intakeshooter?

Reply:
not as a requirement but it still uses it

Reply:
oh wait never mind